template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/profilermanager/ProfilerMetricsRoleConfigProviderTest.java

```diff
@@ -52,27 +54,29 @@ public void testGetRoleConfigsWithSingleRolesPerHostGroup() {
 
```

Comment:
I cannot find any testcase where SSL is enabled, and also the JDBC URL property is not checked anywhere.